docs/habitat.md

```diff
@@ -4,14 +4,18 @@ title: Chef InSpec Integration with Chef Habitat
 
 # Chef Habitat Integration
 
-Chef InSpec provides an easy method to create an executable Chef Habitat package for an Chef InSpec profile. When run via the Chef Habitat Supervisor, the package will run Chef InSpec with your profile and write out its findings to a JSON file. This provides the ability to ship your compliance controls alongside your Chef Habitat-packaged application and continuously run InSpec, providing you *Continuous Compliance.*
+Chef InSpec provides an easy method to create an executable Chef Habitat package for an Chef InSpec profile. When run via the Chef Habitat Supervisor, the package will run Chef InSpec with your profile and write out its findings the supervisor log. This provides the ability to ship your compliance controls alongside your Chef Habitat-packaged application and continuously run InSpec, providing you *Continuous Compliance.*
 
 ## What is Chef Habitat
 
 Chef Habitat by Chef is our new Application Automation tool that aims to make it easy, safe, and fast to build, deploy, and manage applications. From build dependencies, runtime dependencies, dynamic configuration, and service discovery (just to name a few), Chef Habitat packages the automation with the application instead of relying on an underlying platform.
 
 To learn more about Chef Habitat and try our demos and tutorials, visit [https://www.habitat.sh](https://www.habitat.sh).
 
+## Chef Habitat Scaffolding
+
+Chef Habitat has a native templating method to it called scaffolding. This plugin now uses scaffolding so that you can get the benefits of testing and best practices for your InSpec profile Habitat packages. This will also remove the need for you to re-run the plugin over time as things in InSpec may change. In order to pull in the latest changes you just have to re-run build and it will pull in the latest best practices and the most recent version of InSpec.
```

Comment:
```suggestion
Chef Habitat has a native templating method to it called scaffolding. This plugin uses scaffolding so that you can get the benefits of testing and best practices for your InSpec profile Habitat packages. This will also remove the need for you to re-run the plugin over time as things in InSpec may change. With scaffolding, in order to pull in the latest changes you just have to re-run build and it will pull in the latest best practices and the most recent version of InSpec.
```